net-p2p/resilio-sync/resilio-sync-2.6.1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles unpacker user
+
+QA_PREBUILT="usr/bin/rslsync"
+BASE_URI="http://linux-packages.resilio.com/${PN}/deb/pool/non-free/r/${PN}/${PN}_${PV}-1_@arch@.deb"
+
+DESCRIPTION="Resilient, fast and scalable file synchronization tool"
+HOMEPAGE="https://resilio.com/"
+SRC_URI="amd64? ( ${BASE_URI/@arch@/amd64} )
+	x86? ( ${BASE_URI/@arch@/i386} )"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="pax_kernel"
+RESTRICT="mirror"
```

Comment:
Don't you think it needs `RESTRICT=bindist` as well?

Reply:
Added.